public/app/features/transformers/docs/content.ts

```diff
@@ -628,6 +628,52 @@ Use this transformation to construct a matrix by specifying fields from your que
   `;
     },
   },
+  groupToNestedTable: {
+    name: 'Group to nested table',
+    getHelperDocs: function () {
+      return `
+  Use this transformation to group the data by a specified field (column) value, process calculations on each group, and have records that share the same grouped field value to be displayed in a nested table.
+
+  Click to see a list of calculation choices. For information about available calculations, refer to [Calculation types][].
```

Comment:
Click what? When this is in the docs, it's not clear what we're talking about here.

Reply:
That's fair! I moved this over from `Group by` docs so once this is merged can updated there as well 😄

Reply:
Restructured this and added images 😸